pitest-html-report/src/main/java/org/pitest/mutationtest/report/html/MutationTotals.java

```diff
@@ -49,13 +49,13 @@ public void addMutationsDetetcted(final long mutationsKilled) {
   }
 
   public int getLineCoverage() {
-    return this.numberOfLines == 0 ? 100 : Math
-        .round((100f * this.numberOfLinesCovered) / this.numberOfLines);
+    return this.numberOfLines == 0 ? 100 : (int) Math
+        .floor((100f * this.numberOfLinesCovered) / this.numberOfLines);
```

Comment:
you don't even need to do a floor. if you cast the result into an int, it will "floor" it also. But maybe the floor is more explicit on what it is done here.

(int) ((100f * this.numberOfLinesCovered) / this.numberOfLines);